src/wolfgame.py

```diff
@@ -6562,6 +6563,14 @@ def relay(cli, nick, chan, rest):
     if rest.startswith("\u0001PING"):
         cli.notice(nick, rest)
         return
+    if rest == "\u0001VERSION\u0001":
+        try:
+            ans = subprocess.check_output(["git", "log", "-n", "1", "--pretty=format:%h"])
+            reply = "\u0001VERSION lykos {0}, Python {1} -- https://github.com/lykoss/lykos\u0001".format(str(ans.decode()),platform.python_version())
+        except:
```

Comment:
except too broad, should only catch exceptions from subprocess.check_output, e.g. CalledProcessError